src/examples/cancel/Shootist.java

```diff
@@ -56,18 +76,12 @@
 
     private boolean requestTerminated;
 
-    private static Logger logger = Logger.getLogger(Shootist.class);
-
-
-
-
-
     public void processRequest(RequestEvent requestReceivedEvent) {
         Request request = requestReceivedEvent.getRequest();
         ServerTransaction serverTransactionId = requestReceivedEvent
                 .getServerTransaction();
 
-        logger.info("\n\nRequest " + request.getMethod()
+        LOG.info("\n\nRequest " + request.getMethod()
```

Comment:
i bet you refer to test classes that were configuring log4j, that is fine but those should be less. The logger var rename basically changed all the files (at least all classes using logging) which was too much

Regarding code-style and mavenization I totally agree. Mavenization is first item on our techDebt backlog. We are doing some efforts on this, like allowing testSuite to be run in parallel, and we will release this soon. Things like using checkstyle or enforcer plugins are considered alreadt, im happy you are aligned there. Again we may cover this in separate issue, are you interested in contributing on that side (mavenization)?